src/test/c_api_stress_tests.cpp

```diff
@@ -64,6 +66,86 @@ class StressModelCapiConfigChanges : public StressCapiConfigChanges {
     }
 };
 
+TEST_F(ConfigChangeStressTestSingleModel, ChangeToEmptyConfigInference) {
+    bool performWholeConfigReload = true;  // we just need to have all model versions rechecked
+    std::set<StatusCode> requiredLoadResults = {
+        StatusCode::OK};  // we expect full continuity of operation
+    std::set<StatusCode> allowedLoadResults = {
```

Comment:
Since end result of test is that model is unloaded then MODEL_VERSION_NOT_LOADED_ANYMORE is required state.

Reply:
The same applies to test below.